jaxrs/WFLY-11242_JBEAP-15722_built-in_multivalue_param.adoc

```diff
@@ -0,0 +1,92 @@
+= Provide builtin ParamConverter* classes for multi valued parameters
+:author:            Ron Sigal
+:email:             rsigal@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          resteasy
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/JBEAP-15722
+ 
+== Overview
+The JAX-RS specification, implemented by RESTEasy, provides for ParamConverter<T> classes that can
+convert T objects to and from strings. For example, a FooParamConverter, designed to translate Foo objects
+to and from strings, could be used to convert the HTTP header
+----
+f: <foo representation>
+----
+to a Foo object and pass it to the resource method
+--------------------------
+@Path("header")
+@GET
+public String header(@HeaderParam("f") Foo foo) { ... }
+--------------------------
+
+RESTEasy goes beyond the JAX-RS specification by supporting ParamConverters which handle lists, sets, and arrays.
+For example, FooListParamConverter might be designed to convert HTTP header
+--------------------------
+fs: <foo representation 0>,<foo representation 1>
+--------------------------
+
+for  
+
+----
+@Path("header/list")
+@GET
+public String header(@Separator(",") @HeaderParam("fs") List<Foo> foos) { ... }
+----
+
+Out of the box, RESTEasy has not provided any multi-element ParamConverters. However, work in progress for
```

Comment:
s/work in progress for RESTEASY-1996 "Provide builtin ParamConverter* classes for multi valued params"/thisRFE/g